src/main/java/net/phptravels/gui/pages/HomePage.java

```diff
@@ -259,4 +254,8 @@ public VisaFormPage visaFillForm(String fromCountry, String toCountry, String da
 
         return new VisaFormPage(driver);
     }
+    public LoginPage loginForm(){
```

Comment:
not cleat method name
action methods should be named as click***(), type***(), select***() etc
boolean presence verifications method should be named is***Present(), is***SHown() etc